server/package.json

```diff
@@ -2,22 +2,33 @@
   "dependencies": {
     "bcryptjs": "^2.4.3",
     "cookie-parser": "^1.4.6",
-    "cors": "^2.8.5",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "jsonwebtoken": "^9.0.2",
-    "mongoose": "^8.0.1",
-    "nodemon": "^3.0.1"
+    "mongoose": "^8.0.1"
+  },
+  "devDependencies": {
+    "jest": "^29.7.0",
+    "mongodb-memory-server": "^9.1.4",
+    "nodemon": "^3.0.1",
+    "supertest": "^6.3.3"
```

Comment:
can u explain what's this used for ?

Reply:
its a library for http testing you can see it as request(your app/server).get/post/delete(api path) and then your regular jest testing